src/components/document-list.jsx

```diff
@@ -131,7 +131,7 @@ class DocumentList extends React.Component {
   renderZeroState() {
     const editableClass = !this.props.isEditable ? 'disabled' : '';
 
-    if (this.props.docs.length === 0) {
+    if (this.props.status === 'active' && this.props.docs.length === 0) {
```

Comment:
What do you thinking about changing this to use the return early pattern?

```
renderZeroState() {
    if (this.props.docs.length > 0 && this.props.status !== 'active') {
      return null;
    }
    const editableClass = !this.props.isEditable ? 'disabled' : '';
    // rest of existing render() function body 
...
```

Instead of:

```
renderZeroState() {
    const editableClass = !this.props.isEditable ? 'disabled' : '';

    if (this.props.docs.length === 0) {
    if (this.props.status === 'active' && this.props.docs.length === 0) {
    // rest of existing render() function body 
...
```

Reply:
I can use return early pattern here, but it should be i think:

```
if (this.props.docs.length > 0 || this.props.status !== 'active') {
      return null;
}
```

Reply:
oop! sorry for the typo